tests/integration/cli/test_rituals.py

```
@@ -132,6 +209,140 @@ def test_unknown_ritual_is_a_usage_error(tmp_path, monkeypatch, capsys):
         assert "no ritual named 'nope'" in capsys.readouterr().err
 
 
+@pytest.mark.usefixtures("_home")
+class TestRitualPackages:
+    @staticmethod
+    def test_a_package_is_found_and_lists_every_ritual(tmp_path, monkeypatch, capsys):
```

Ruff (0.16.1)

[warning] Missing return type annotation for staticmethod `test_a_package_is_found_and_lists_every_ritual`

Add return type annotation: `None`

(ANN205)